api/pkg/apis/v1alpha1/providers/stage/create/create.go

```diff
@@ -185,7 +192,7 @@ func (i *CreateStageProvider) Process(ctx context.Context, mgrContext contexts.M
 					create,
 					functionName,
 					metrics.ProcessOperation,
-					metrics.DeleteOperationType,
```

Comment:
So we don't distinguish delete or udpate anymore?

Reply:
For stage provider, we can distinguish operation type. But considering target provider, it may consists of multiple update + remove operations, I am thinking it is better we could calculate per function level.